app/gcl/cloud-digital-leader/section3/components/sections/Section3_BQML.tsx

```
@@ -0,0 +1,96 @@
+                        </table>
+                    </div>
+                </div>
+
+                <div className={baseStyles.card} style={{ background: 'var(--color-bg-secondary)' }}>
+                    <div style={{ fontFamily: 'var(--font-mono)', fontSize: '1rem', color: 'var(--color-text-muted)', marginBottom: '0.8rem' }}>
+                        SQL — BigQuery ML モデル作成の例
+                    </div>
+                    <div className="code-block" style={{ fontFamily: 'var(--font-mono)', fontSize: '1rem', color: 'var(--color-text-secondary)', overflowX: 'auto', lineHeight: 1.8 }}>
+                        <div className="code-line"><span style={{ color: '#7c4dff' }}>CREATE OR REPLACE MODEL</span> <span style={{ color: '#00bcd4' }}>`myproject.mydataset.churn_model`</span></div>
+                        <div className="code-line"><span style={{ color: '#7c4dff' }}>OPTIONS</span>(</div>
+                        <div className="code-line">  model_type = <span style={{ color: '#fbbc04' }}>'logistic_reg'</span>,</div>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)